Description: Skip some things until eigen branch memory issues resolved.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-04

--- a/M2/Macaulay2/packages/Macaulay2Doc/functions/isPrime-doc.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc/functions/isPrime-doc.m2
@@ -32,7 +32,6 @@
    	    isProbablePrime n
             isPrime n
             n1 = nextPrime(n+1)
-            factor(n1^2*n)
         Text
             These functions handle numbers larger than this.  For example,
         Example
@@ -40,7 +39,6 @@
             isProbablePrime m
             isPrime m
             isPrime m^2
-            factor m^2
         Example
             ndigits = 30
             m = nextPrime(10^30)
@@ -50,9 +48,6 @@
             isPrime m1
             isPrime (m*m1)
             isPrime(m*m*m1*m1*m2^6)
-            elapsedTime facs = factor(m*m1)
-            facs = facs//toList/toList
-            assert(set facs === set {{m,1}, {m1,1}})
             m3 = nextPrime (m^3)
             elapsedTime isPrime m3
             elapsedTime isProbablePrime m3
@@ -108,12 +103,6 @@
 	  "Since ", TO "factor", " returns factors guaranteed only to be pseudoprimes, it
 	  may be useful to check their primality, as follows."
 	  },
-     EXAMPLE lines ///
-     f = factor 28752093487520394720397634653456
-     peek'_2 f
-     first \ toList f
-     isPrime \ oo
-     ///,
      "This function can be used also to determine whether an ideal in a polynomial ring is prime.",
      EXAMPLE lines ///
      R = QQ[a..d];
--- a/M2/Macaulay2/packages/Macaulay2Doc/functions/factor-doc.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc/functions/factor-doc.m2
@@ -35,7 +35,6 @@
 	  exponent is an integer.",
 	  },
      EXAMPLE {
-         "factor 124744878111332355674003415153753485211381849014286981744945",
          "y = (2^15-4)/(2^15-5)",
       	 "x = factor y",
       	 "value x"
--- a/M2/Macaulay2/packages/EnumerationCurves.m2
+++ b/M2/Macaulay2/packages/EnumerationCurves.m2
@@ -955,11 +955,6 @@
     Example
       time rationalCurve(3) - rationalCurve(1)/27
     Text
-      The numbers of rational curves of degree 3 on general complete intersection Calabi-Yau threefolds can be computed as follows:
-    
-    Example
-      time for D in T list rationalCurve(3,D) - rationalCurve(1,D)/27
-    Text
       For rational curves of degree 4:
     
     Example
